### ISSUE TITLE
quant_noise.py, process for  kernel_size==(1,1) change input weight size ,cause error

### ISSUE THREAD

**Issue by misslibra** [pytorch/fairseq#2207]
## 🐛 Bug

`
                if mod.kernel_size == (1, 1):
                    print('in_features // block_size * out_features is :', in_channels // block_size * out_channels)
                    mask = torch.zeros(int(in_channels // block_size * out_channels), device=weight.device)
                    print('mask shape 1 : ', mask.shape)
                    mask.bernoulli_(p)
                    mask = mask.repeat_interleave(block_size, -1).view(-1, in_channels)
                    print('1*1 mask shape ', mask.shape, block_size, in_channels)
                else:
                    mask = torch.zeros(weight.size(0), weight.size(1), device=weight.device)
                    mask.bernoulli_(p)
                    print(weight.size(0), weight.size(1), '\n', '3*3 mask shape 0 : ', mask.shape)
                    print('3*3 mask shape 1 : ', mask.shape)

                    mask = mask.unsqueeze(2).unsqueeze(3).repeat(1, 1, mod.kernel_size[0], mod.kernel_size[1])
                    print('3*3 mask shape ', mask.shape, mod.kernel_size[0], mod.kernel_size[1])
`

### Environment

 - fairseq Version ( master):
 - PyTorch Version (1.2)
 - OS (Linux):
 - How you installed fairseq (no install .... put code into my model code):
 - Build command you used (if compiling from source):
 - Python version:3.6
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by pierrestock** [comment]
Hi misslibra, 

Thanks for your interest in our work and for your bugfix! We are planning to release code + models for various vision tasks soon, stay tuned!
